### DISCUSSION TITLE
prognostic aerosol, diagnostic ice ?

### DISCUSSION THREAD

**Discussion by jchylik** [ARM-Development/comble-mip#32] [General]
One product expected from Part  II is "prognostic aerosol, diagnostic ice"
What does it mean? Unfortunately I cannot find more description of this anywhere on the page,

Does it mean that the activation spectrum of ice nucleating particles would be described by 1-moment or 2-moment scheme, but the mass partitioning between ice and water would be calculated just based on temperature?

**Comment by twjuliano** [comment]
Hi Jan,

Thanks for your question. We have updated the "Aerosol & microphysics" part of the [Main Model Configuration page](https://arm-development.github.io/comble-mip/main_configuration.html) to hopefully clarify our specifications. In short, the simulation in question ("prognostic aerosol, diagnostic ice") would apply the descriptions under "Diagnostic ice formation" and "Prognostic aerosol".

Please let us know if you have any further questions about this.

Tim
